### ISSUE TITLE
Use dali.plugin.tf  error

### ISSUE THREAD

**Issue by jxmelody** [NVIDIA/DALI#52]
- CentOS Linux release 7.5.1804 
- Python 2.7.5
- tensorflow-gpu 1.9
-  GeForce GTX 1080 * 2

I used the floowing command to install nvidia.dali:

```
pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali
```

But when use `dali.plugin.tf` ,I got the error:
```
>>> import nvidia.dali.plugin.tf as dali_tf
>>> daliop=dali_tf.DALIIterator()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python2.7/site-packages/nvidia/dali/plugin/tf.py", line 20, in DALIIterator
    dali_tf_module = tf.load_op_library(libdali_tf)
  File "/usr/lib/python2.7/site-packages/tensorflow/python/framework/load_library.py", line 56, in load_op_library
    lib_handle = py_tf.TF_LoadLibrary(library_filename)
tensorflow.python.framework.errors_impl.NotFoundError: /usr/lib64/python2.7/site-packages/nvidia/dali/plugin/libdali_tf.so: undefined symbol: _ZN9perftools8gputools6Stream18BlockHostUntilDoneEv
```

**Comment by Kh4L** [comment]
Hi @jxmelody !
This seems to be a mismatch of the TensorFlow 1.9 ABI/API.
For the moment, our TensorFlow plugin is built against tensorflow-gpu 1.8.
This will be fixed really soon !

**Comment by jxmelody** [comment]
Hi @Kh4L ,
I thought it would be a mismatch too, and I‘am going to uninstall TensorFlow 1.9, re-install TensorFlow 1.8 as well...
Thanks !

**Comment by JanuszL** [comment]
Lets leave it open. I just created "Release_0.1.1+" milestone and let us gather all issues which are going to be fixed by next release.

**Comment by cliffwoolley** [comment]
Fix is in 0.1.2 in https://github.com/NVIDIA/DALI/commit/b7a108491259a91b2dcdba5eeef91a1de08ae135 .
